### ISSUE TITLE
IA Feedback - Design intent - Decision Tools - Sitewide Search UI Consistency

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/va.gov-team#23538]
## VFS acceptance criteria
- [ ] Let VSP know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Wondering what the thought/rationale was for the H1 and breadcrumb change from "Find a VA Form" to "Find VA Forms"?

A few items that would not be considered a "UI refresh", but for future reference:
- There are a number of differences in how each of these search tools were built in terms of page flow - i.e. some results are displayed on a page refresh (global search, find forms), others are displayed on a new page (R&S, yellow ribbon shool search) - this impacts a few things:  the breadcrumb and whether users can get back to the original display state of a page, a change in the H1 to indicate a content refresh for a user, etc. 
- Somewhat unrelated, but related, to this work...the va.gov/search URL is redirecting to the home page.  I don't recall the reason why this is, but this redirect should be removed.  It would not be a high traffic page, but user's should be able to access a default display state of search.  This change should also impact the breadcrumb by adding in a default state to the path:  Home > Search > Search results

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
- Breadcrumbs are inaccurate in prototypes.  Make sure that the desktop breadcrumb shows the complete path including the current page.  The mobile breadcrumb should only show a link to the parent page.  
  - Global search results desktop:  Home > Search results
  - Global search results mobile:  < Home
  - Resources search results desktop:  Home > Resources and support > Search results
  - Resources search results mobile:  < Resources and support
  - Forms search results desktop:  Home > Find a VA Form
  - Forms search results mobile:  < Home
- The H1 for Resources search results page needs to indicate the user is on a new page.  The H1 should remain as "Search results" or something similar to indicate they are on a different page and ensure the desktop breadcrumb is not redundant (i.e. Home > Resources and support  > Resources and support).  
- Some of the links within the right rail of the global search are there because of stakeholder requests (i.e. the BVA lookup).  Appropriate conversations/investigation must take place to before these links are considered for removal.  That said, I have no issues with exploring different placement/design ideas for these links in support of deprecating the right rail.  :-)

**Should:**
- The long intro copy impacts the experience on mobile devices on the search results for the resources and forms tools.  For the forms search tool, the page is a refresh, so we reduce the length of that intro copy on both the default state and the search results.  For resources and support, we currently do not show the long content on the results page, and it should be tested/validated by users before its changed to ensure it does not negatively impact their experience. 

**Consider:**
- An option for consistency between H1s on these search tools is to use a similar structure for the H1s on the results pages.  Examples: 
  - Global search:  "Search results for VA.gov" or "VA.gov search results"
  - Resources:  "Search results for resources and support" or "Resources and support search results"
  - Find a VA form: "Search results for VA forms" or "Find a VA form search results"

**Comment by mnorthuis** [comment]
Please let me know if you have any questions about the information above, some of it is hard to describe across all the tools.  Happy to collaborate as needed.  Thank you!

**Comment by MarciMcGuireGCIO** [comment]
Thank you, @mnorthuis!  @callen2563 and @mgleo please let me know once you've had a chance to review so we can discuss next steps.

**Comment by MarciMcGuireGCIO** [comment]
@mnorthuis, @joshkimux, @1Copenut -- I am probably mis-interpreting, but the feedback related to the mobile breadcrumb above ("The mobile breadcrumb should only show a link to the parent page.") seems contradictory to the Accessibility feedback related to the mobile breadcrumb in this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/23387.   ("Fix mobile breadcrumbs so that they are all consistent with one another. Find a Form and Resources and Support should have the same type of breadcrumb as search e.g. "Home > Find a Form")

Can you please let us know which direction we should take?  Thank you! 😄 

cc @callen2563 @mgleo

**Comment by mnorthuis** [comment]
The initial feedback in the accessibility ticket isn't quite accurate to how our breadcrumbs work today.  Mobile breadcrumbs only show the previous page.  Trevor clarifies this with his comment in that ticket:  https://github.com/department-of-veterans-affairs/va.gov-team/issues/23387#issuecomment-822818505.    What I have typed above for breadcrumbs is accurate based on page flow/structure and H1s as they are today.   Let me know if that helps!

**Comment by MarciMcGuireGCIO** [comment]
Thank you, @mnorthuis!  That is very helpful!

**Comment by callen2563** [comment]
@mnorthuis @MarciMcGuireGCIO I have updated the FAF and R+S mock ups here:

[Mobile updates](https://vsateams.invisionapp.com/overview/UI-Search-Consistency---mobile-cknr2uisl2tl001ymf60mfq36/screens?sortBy=1&sortOrder=1&viewLayout=2)

[Desktop updates](https://vsateams.invisionapp.com/overview/UI-Search-Consistency---desktop-cknr2uisl2tl101ymmt48cx3z/screens?sortBy=1&sortOrder=1&viewLayout=2)

_"An option for consistency between H1s on these search tools is to use a similar structure for the H1s on the results pages. Examples:
Global search: "Search results for VA.gov" or "VA.gov search results"
Resources: "Search results for resources and support" or "Resources and support search results"
Find a VA form: "Search results for VA forms" or "Find a VA form search results"_

Note - I hope I placed the search results copy in the location you had in mind, let me know if I should move it.

There have been some updates on the breadcrumbs [discussed here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/23387) 

Let me know if I missed any thing. Thanks! Cassandra
